packages/server/src/constants/automations.ts

```diff
@@ -0,0 +1,6 @@
+import { Duration } from "@budibase/backend-core"
+import env from "../environment"
+
+const defaults = env.getDefaults()
+export const AUTOMATION_SYNC_TIMEOUT = Duration.fromMinutes(2).toMs()
+export const AUTOMATION_ASYNC_TIMEOUT = defaults.QUERY_THREAD_TIMEOUT
```

Comment:
This seems a bit weird. So if someone overrides `QUERY_THREAD_TIMEOUT`, this file wouldn't pick it up? It would be whatever the default is?

Reply:
Got rid of this file and just used the defaults directly everywhere - no point in really duplicating them like this.